lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -151,10 +151,10 @@ defmodule ExUnit.DocTest do
     defexception [:message]
 
     def exception(opts) do
-      module  = Keyword.fetch!(opts, :module)
+      module = Keyword.fetch!(opts, :module)
       message = Keyword.fetch!(opts, :message)
 
-      file = module.__info__(:compile)[:source] |> Path.relative_to_cwd
+      file = module.__info__(:compile)[:source] |> Path.relative_to_cwd()
```

Comment:
In this case it's obvious that `Path.relative_to_cwd` is a function call. Why the parens?

Reply:
The formatter standardizes all functions calls to have parens.